src/app/twitch-manage-users/twitch-manage-users.component.ts

```diff
@@ -5,16 +5,25 @@ import { AngularFire, FirebaseListObservable } from 'angularfire2';
 @Component({
   selector: 'app-twitch-manage-users',
   templateUrl: './twitch-manage-users.component.html',
-  styleUrls: ['./twitch-manage-users.component.css']
+  styleUrls: ['./twitch-manage-users.component.css'],
+  
 })
 export class TwitchManageUsersComponent implements OnInit {
 
   constructor(private service: TwitchDatabaseService) { }
 
     userNames: FirebaseListObservable<any> = this.service.getUsers();
-
+    newUser: string;
   ngOnInit() {
     console.log(this.userNames);
   }
 
+  addNewUser(un: string){
```

Comment:
you can just do addNewUser() - no need to pass the username in, it is already in scope, you can just refer to this.userName within this function